notebooks/advanced_topics/IDC_Recipes_Conversion.ipynb

```diff
@@ -0,0 +1,2064 @@
+    "outputId": "5c645fc7-9753-4dfc-d178-cfbfde90d57e"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "cp s3://idc-open-data/6d7f4ec7-2c84-4a46-86ac-acde279195bb/49cbd3fa-638d-4da2-9432-54d71fcf7dd0.dcm rgb-dicom/49cbd3fa-638d-4da2-9432-54d71fcf7dd0.dcm\n",
+      "cp s3://idc-open-data/6d7f4ec7-2c84-4a46-86ac-acde279195bb/d2031780-db50-49bc-b2b0-186213fedb34.dcm rgb-dicom/d2031780-db50-49bc-b2b0-186213fedb34.dcm\n",
+      "cp s3://idc-open-data/6d7f4ec7-2c84-4a46-86ac-acde279195bb/74527996-46d8-4f42-af90-4b87c41a66f1.dcm rgb-dicom/74527996-46d8-4f42-af90-4b87c41a66f1.dcm\n",
+      "cp s3://idc-open-data/6d7f4ec7-2c84-4a46-86ac-acde279195bb/e0076901-151f-43dd-8f56-9e2adec521da.dcm rgb-dicom/e0076901-151f-43dd-8f56-9e2adec521da.dcm\n",
+      "cp s3://idc-open-data/6d7f4ec7-2c84-4a46-86ac-acde279195bb/5ee2770f-b222-4565-a3bd-46ba294ca222.dcm rgb-dicom/5ee2770f-b222-4565-a3bd-46ba294ca222.dcm\n",
+      "--2024-06-06 16:09:37--  https://openslide.cs.cmu.edu/download/openslide-testdata/Aperio/CMU-1-Small-Region.svs\n",
+      "Resolving openslide.cs.cmu.edu (openslide.cs.cmu.edu)... 128.2.209.93\n",
+      "Connecting to openslide.cs.cmu.edu (openslide.cs.cmu.edu)|128.2.209.93|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 1938955 (1.8M) [image/tiff]\n",
+      "Saving to: ‘rgb-svs/CMU-1-Small-Region.svs’\n",
+      "\n",
+      "CMU-1-Small-Region. 100%[===================>]   1.85M  1.51MB/s    in 1.2s    \n",
+      "\n",
+      "2024-06-06 16:09:38 (1.51 MB/s) - ‘rgb-svs/CMU-1-Small-Region.svs’ saved [1938955/1938955]\n",
+      "\n",
+      "rgb-dicom/:\n",
+      "total 2781536\n",
+      "drwxr-xr-x@  7 erindiel  staff   224B Jun  6 16:09 \u001b[34m.\u001b[m\u001b[m\n",
+      "drwxr-xr-x  10 erindiel  staff   320B Jun  6 16:09 \u001b[34m..\u001b[m\u001b[m\n",
+      "-rw-r--r--@  1 erindiel  staff   6.3M Jun  6 16:09 49cbd3fa-638d-4da2-9432-54d71fcf7dd0.dcm\n",
+      "-rw-r--r--@  1 erindiel  staff   1.1G Jun  6 16:09 5ee2770f-b222-4565-a3bd-46ba294ca222.dcm\n",
+      "-rw-r--r--@  1 erindiel  staff   1.6M Jun  6 16:09 74527996-46d8-4f42-af90-4b87c41a66f1.dcm\n",
+      "-rw-r--r--@  1 erindiel  staff    19M Jun  6 16:09 d2031780-db50-49bc-b2b0-186213fedb34.dcm\n",
+      "-rw-r--r--@  1 erindiel  staff   177M Jun  6 16:09 e0076901-151f-43dd-8f56-9e2adec521da.dcm\n",
+      "\n",
+      "rgb-svs/:\n",
+      "total 4648\n",
+      "drwxr-xr-x@  3 erindiel  staff    96B Jun  6 16:09 \u001b[34m.\u001b[m\u001b[m\n",
+      "drwxr-xr-x  10 erindiel  staff   320B Jun  6 16:09 \u001b[34m..\u001b[m\u001b[m\n",
+      "-rw-r--r--@  1 erindiel  staff   1.8M Dec  6  2013 CMU-1-Small-Region.svs\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Download sample data from IDC\n",
+    "!s5cmd --no-sign-request --endpoint-url https://s3.amazonaws.com cp \"s3://idc-open-data/6d7f4ec7-2c84-4a46-86ac-acde279195bb/*\" rgb-dicom\n",
```

Comment:
Instead, do this:

```
!idc download-from-selection --series-instance-uid 1.3.6.1.4.1.5962.99.1.3140643155.174517037.1639523215699.2.0 --download-dir ./rgb-dicom --dir-template ""
```

This will report the total size and check that you have enough size, and will report download progress.